src/Infrastructure/Builder/Yaml/LifecycleEventDataBuilderInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Builder\Yaml;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\InitializedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\RemovedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\UpdatedEventData;
+
+interface LifecycleEventDataBuilderInterface
+{
+    /**
+     * @param array $taskData
+     * @param array $taskListData
+     * @param array $tags
+     *
+     * @return \SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\InitializedEventData
+     */
+    public function buildInitializedEventData(array $taskData, array $taskListData, array $tags = []): InitializedEventData;
```

Comment:
Please use DTO instead. In this case, would be possible to combine it under one method and make it similar to the other builders